test.js

```diff
@@ -174,6 +174,9 @@ test('no format', function() {
   assert.equal(fecha.format(new Date(2017,4,2,10)), 'Tue May 02 2017 10:00:00');
 });
 
+testFormat('literal in format', new Date(2001, 2, 5, 6, 7, 2, 5), '[on] MM-DD-YYYY [at] HH:mm',
+  'on 03-05-2001 at 06:07');
+
```

Comment:
This is a good start on tests, but I would like to see a bunch more. Can you port over the tests from moment?
https://github.com/moment/moment/blob/develop/src/test/moment/format.js#L12